sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-card>
```

Comment:
I see that the `AssessmentsSummary` page uses the `<full-page-container>`. Is there a specific reason why we might not be using it here?